clients/pkg/promtail/targets/manager.go

```diff
@@ -241,6 +244,12 @@ func NewTargetManagers(
 				return nil, errors.Wrap(err, "failed to make kafka target manager")
 			}
 			targetManagers = append(targetManagers, kafkaTargetManager)
+		case AzureEventHubScrapeConfigs:
+			kafkaTargetManager, err := azureeventhub.NewTargetManager(reg, logger, client, scrapeConfigs)
```

Comment:
super minor nit: I would change this name to avoid any issues when scanning the code.